src/lib/ng-package/entry-point/write-package.transform.ts

```diff
@@ -35,55 +35,81 @@ export const writePackageTransform = (options: NgPackagrOptions) =>
     ];
 
     if (!ngEntryPoint.isSecondaryEntryPoint) {
-      const assetFiles: string[] = [];
-
       // COPY ASSET FILES TO DESTINATION
       spinner.start('Copying assets');
 
       try {
-        for (const asset of [...ngPackage.assets, 'LICENSE', '**/README.md']) {
-          let assetFullPath = path.join(ngPackage.src, asset);
-
+        type AssetCopyTask = [srcPath: string, destPath: string];
+        const assetCopyTasks: AssetCopyTask[] = [];
+
+        // Parse string assets and add to tasks
+        const assetGlobs = ngPackage.assets
+          .filter((asset): asset is string => typeof asset === 'string')
+          .concat('LICENSE', '**/README.md');
+        for (const assetGlob of assetGlobs) {
+          const assetFullGlob = path.join(ngPackage.src, assetGlob);
+          let assetFullGlobIs: 'file' | 'dir' | null = null;
           try {
-            const stats = await stat(assetFullPath);
-            if (stats.isFile()) {
-              assetFiles.push(assetFullPath);
-              continue;
-            }
-
-            if (stats.isDirectory()) {
-              assetFullPath = path.join(assetFullPath, '**/*');
-            }
+            const stats = await stat(assetFullGlob);
+            const isFile = stats.isFile() || assetGlob === 'LICENSE';
+            const isDir = stats.isDirectory();
+            assetFullGlobIs = isFile ? 'file' : isDir ? 'dir' : null;
```

Comment:
We should just use `isFile` and `isDir`.

Reply:
Done